src/components/pages/Profile/Header/index.tsx

```diff
@@ -35,7 +37,9 @@ export default function AccountHeader({
     <div className={styles.grid}>
       <div>
         <Account accountId={accountId} />
-        <Stats accountId={accountId} />
+        {appConfig?.allowDynamicPricing === 'true' && (
+          <Stats accountId={accountId} />
```

Comment:
I would like to have this more specific:
- Always show stats
- Only show `liquidity`stats when allowDynamicPricing = 'true'

So you can always see stats like "Sales" or "Published" but the below screenshot is only visible when dynamic pricing is allowed:
![grafik](https://user-images.githubusercontent.com/28757404/142668397-5044d838-a8bd-4390-b65d-dba6fff75caa.png)

Reply:
Since the Publish tab is only showed for whitelisted addresses, should I check on this page as well? Otherwise isn't the number for "published" and "sales" always going to be zero?